Makefile

```diff
@@ -27,6 +27,10 @@ ifeq ($(DOCKER_BUILDKIT),)
 export DOCKER_BUILDKIT ?= 1
 endif
 
+ifeq ($(MULTI_ARCH),)
```

Comment:
What didn't work for you? I don't think we need this in the Makefile. MULTI_ARCH was just for Jenkins / csm-shared-library.